govc/USAGE.md

```diff
@@ -324,8 +324,6 @@ but appear via `govc $cmd -h`:
  - [vm.vnc](#vmvnc)
  - [volume.ls](#volumels)
  - [volume.rm](#volumerm)
- - [vsan.change](#vsanchange)
```

Comment:
Looks like your govc binary is out of date, vsan.change and vsan.info were recently added. Same for the vm.change flags.
Realizing the script (govc/usage.sh) uses govc in `$PATH`, we can fix that up later to use govc from the build tree.